components/alert/src/Alert.js

```diff
@@ -0,0 +1,128 @@
+/* @jsx jsx */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme, paint } from '@westpac/core';
+
+import { Button } from '../../button/src';
+import { CloseIcon } from '../../icon/src';
+
+import { CSSTransition } from 'react-transition-group';
+
+// ==============================
+// Component
+// ==============================
+
+export const Alert = ({ appearance, closable, icon: Icon, children }) => {
+	const [open, setOpen] = useState(true);
+	const { breakpoints, alert } = useTheme();
+	const mq = paint(breakpoints);
+
+	const styleCommon = {
+		marginBottom: alert.marginBottom,
+		padding: alert.padding,
+		paddingRight: [18, 30],
+		position: 'relative',
+		display: ['block', 'flex'],
+		zIndex: 1,
+		transition: 'opacity 0.3s ease-in-out',
+
+		a: {
+			color: 'inherit',
+		},
+
+		'&.anim-exit-active': {
+			opacity: 0,
+		},
+	};
+
+	const styleAppearance = {
+		...alert.appearance[appearance],
```

Comment:
I would simplify the alert tokens to just color, backgroundColor and borderColor. Then set up the components appearance styling as...
```
const styleAppearance = {
	color: alert.appearance[appearance].color,
	backgroundColor: alert.appearance[appearance].backgroundColor,
	borderTop: `${alert.borderWidth} solid`,
	borderBottom: `${alert.borderWidth} solid`,
	borderColor: alert.appearance[appearance].borderColor,
};
```

with token shape...
```
success: {
	color: COLORS.success,
	backgroundColor: tint(COLORS.success, 0.1),
	borderColor: tint(COLORS.success, 0.5),
},
```

The alert borders are always going to be the same border width top and bottom, and always solid border style. The tokens don't need this granularity.